template/en/default/bug/dependency-graph.html.tmpl

```diff
@@ -53,7 +53,7 @@
 </p>
 
 [% IF image_map %]
-  <img src="[% image_url %]" alt="Dependency graph" usemap="#imagemap">
+  <img src="data:image/png;base64,[% image_data FILTER html %]" alt="Dependency graph" usemap="#imagemap">
```

Comment:
There's no need to `FILTER html` here as we know the content is base64; let's do `FILTER none` to avoid the overhead.